src/main/java/com/shank/offcoder/controllers/Controller.java

```diff
@@ -231,11 +231,11 @@ public void attemptLogin(String handle, String password, boolean auto) {
                     dialog.showAndWait();
                     passwordField.setText("");
                 } else {
-                    if (AppData.get().getData("download_number", 0) == 0) {
+                    if (AppData.get().getData("AppData.DOWNLOADED_NUM", 0) == 0) {
                         AppData app = AppData.get();
-                        app.writeData("download_number", 0);
+                        app.writeData("AppData.DOWNLOADED_NUM", 0);
                     } else {
-                        downloadNumber = AppData.get().getData("download_number", 0);
+                        downloadNumber = AppData.get().getData("AppData.DOWNLOADED_NUM", 0);
                     }
                     handleField.setText("");
```

Comment:
Why are the constants mentioned inside double quotes ? @Aqua-16

Reply:
The default didn't give me an error. I forgot to use "" in the replace. I'll fix it in a second.